lib/tasks/db.rake

```diff
@@ -5,6 +5,14 @@ end
 
 # we need to run seed_fu every time we run rake db:migrate
 task 'db:migrate' => ['environment', 'set_locale'] do
+  if Rails.env.production?
+    # Run deferred migrations 30 mins later
+    Jobs.enqueue_in(1800, :run_deferred_migrations)
```

Comment:
I need to write code to figure out which deferred migration has already been run and run those migration.